hw/ip/otbn/dv/otbnsim/sim/isa.py

```diff
@@ -291,11 +291,11 @@ class InstructionBNAQType(InstructionW3Type):
 
 To specify, use the literal syntax .Z""")
     field_wrd_hwsel = Field(name="wrd_hwsel", base=29, size=1, description="")
-    field_wrs1_qwsel = Field(name="wrs1_qwsel",
+    field_wrs2_qwsel = Field(name="wrs2_qwsel",
                              base=27,
                              size=2,
                              description="")
-    field_wrs2_qwsel = Field(name="wrs2_qwsel",
+    field_wrs1_qwsel = Field(name="wrs1_qwsel",
```

Comment:
Could you swap the ordering here? Now we've got `wrs2_qwsel` before `wrs1_qwsel`.

Reply:
I left it that way as it matches the field ordering going from highest bit to lowest bit (so `wrd_hwsel` at 29 then `wrs2_qwsel` at 28-27 and `wrs1_qwsel` at 26-25)

Reply:
Oh, good point.